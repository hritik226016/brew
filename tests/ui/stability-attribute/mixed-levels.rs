//! Test stability levels for items formerly dependent on multiple unstable features.
//@ aux-build:mixed-levels.rs

extern crate mixed_levels;

const USE_STABLE: () = mixed_levels::const_stable_fn();
const USE_UNSTABLE: () = mixed_levels::const_unstable_fn();
//~^ ERROR `const_unstable_fn` is not yet stable as a const fn

fn main() {
    mixed_levels::stable_mac!();
    mixed_levels::unstable_mac!(); //~ ERROR use of unstable library feature `unstable_a` [E0658]
}
