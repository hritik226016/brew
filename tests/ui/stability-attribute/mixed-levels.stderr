error[E0658]: use of unstable library feature `unstable_a`
  --> $DIR/mixed-levels.rs:12:5
   |
LL |     mixed_levels::unstable_mac!();
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add `#![feature(unstable_a)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: `const_unstable_fn` is not yet stable as a const fn
  --> $DIR/mixed-levels.rs:7:26
   |
LL | const USE_UNSTABLE: () = mixed_levels::const_unstable_fn();
   |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add `#![feature(unstable_c)]` to the crate attributes to enable

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
